docs/content/users/basics/faq.md

```diff
@@ -1,144 +1,217 @@
-# Frequently-Asked Questions (FAQ)
+# FAQ
```

Comment:
I suspect the old way catches more SEO and provides more than one way to find this in search.

Reply:
I mentioned search in the first bullet point of the PR description: I kept the phrase “frequently-asked questions” on the page with that in mind. Giving each question a proper heading is *probably* better for overall SEO surfacing those questions—and I’m guessing `ddev faq` would be a more common search than `ddev frequently asked questions`, but that could be verified with research.

The real win would be providing [structured data to further expose Q&A items](https://developers.google.com/search/docs/appearance/structured-data/faqpage), but that’d require a far deeper journey into the theme.